webapp/sources/utils/src/main/scala/zio/syntax.scala

```diff
@@ -5,7 +5,7 @@ import zio.IO
  * .fail and .succeed were removed in RC18
  */
 object syntax {
-  implicit class ToZio[A](a: A) {
+  implicit class ToZio[A](a: => A) {
```

Comment:
~It looks legit, but it doesn't seem to fix the issue, however doing a `ZIO.succeed(...)` explicitly works :exploding_head:~ I had to re-install the `utils` project locally, it works !